src/tools/extended-test-framework/scripts/build.sh

```diff
@@ -5,6 +5,7 @@ set -e pipefail
 SCRIPT_DIR=$(dirname "$0")
 pushd ${SCRIPT_DIR}
 
+pushd ../log_monitoring/scripts && ./build.sh; popd
```

Comment:
```suggestion
pushd ../log_monitor/scripts && ./build.sh; popd
```
Looks like this change is needed for build.sh